test/synchronous/test_collection.py

```diff
@@ -39,13 +39,13 @@
     wait_until,
 )
 
-from bson import encode
+from bson import RawBSONDocument, encode
```

Comment:
Does this file ever get run? Seems like a duplicate of test/test_collection.py

Reply:
This is generated from `test/asynchronous/test_collection.py`, left as an example result for our future test asynchronization work/